<launch>
  <arg name="launch_prefix" default="" />
  <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="node_namespace" default="camera" />
  <arg name="camera_name" default="XPS15_webcam" />
  <arg name="camera_info_url" default="file://$(find aruco_ros_cvwrap)/camera_info/$(arg camera_name).yaml" />
  <arg name="index" default="0"/>
  <arg name="image_topic" default="image"/>
  <node pkg="aruco_ros_cvwrap" type="camera_node" name="$(arg node_namespace)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="camera_name" value="$(arg camera_name)" />
    <param name="camera_info_url" value="$(arg camera_info_url)"/>
    <param name="image_topic" value="$(arg image_topic)" />
    <param name="index" value="$(arg index)" />
  </node>

</launch>
